src/index.html

```diff
@@ -12,7 +12,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
 </head>
-<body>
+<body class="js-enabled">
```

Comment:
Might be worth adding a comment here that this is required for GDS components to be styled correctly. Even better would be to add it in the js so that we definitely know that js-enabled, but I don't know how necessary that is given that the site is an angular app anyway.